model/host/host.go

```diff
@@ -208,6 +208,10 @@ func (h *Host) SetUnprovisioned() error {
 			},
 		},
 	)
+	if err != nil {
+		grip.Error(errors.Wrapf(err, "error unprovisioning host %s", h.Id))
+	}
```

Comment:
don't return error; you can just log the errors.Wrap(err)

Reply:
done